framework/src/main/java/org/tron/core/services/http/solidity/SolidityNodeHttpApiService.java

```diff
@@ -184,7 +184,7 @@ public void start() {
       context.addServlet(new ServletHolder(triggerConstantContractServlet),
           "/walletsolidity/triggerconstantcontract");
 
-      context.addServlet(new ServletHolder(getNodeInfoServlet), "/wallet/getnodeinfo");
+      context.addServlet(new ServletHolder(getNodeInfoServlet), "/monitor/getnodeinfo");
```

Comment:
To be compared with the scope name `wallet`.
`getnodeinfo` may be under the scope of `node`, `system`, `network` or `status`.

> monitor: *n.* An instrument or device used for observing, checking, or keeping a continuous record of a process or quantity.

Reply:
We are changing some apis to a new namespace, this is only a temporary name